src/main/webapp/app/exam/manage/student-exams/student-exam-timeline/programming-exam-diff/programming-exercise-exam-diff.component.html

```diff
@@ -1,7 +1,7 @@
 <h4 class="exercise-title">
     {{ exercise.title }}
     <span
-        >[{{ exercise.maxPoints }} {{ 'artemisApp.examParticipation.points' | artemisTranslate }}]
+        >[{{ exercise.maxPoints }} {{ 'artemisApp.examParticipation.exercisePoints' | artemisTranslate }}]
```

Comment:
We want to get away from using the `artemisTranslate` pipe, and instead use the `jhiTranslate`. You can also pass in values to the String with `translateValues`. If that's possible here, it would be nice

Reply:
Yes exactly, but using jhiTranslate here forces me to change the translation value of exercisePoints, because it uses brackets around the points, where I need to integrate those brackets into the translation value. But this translation key is also used in different parts of Artemis, and they don't use brackets, so I need to introduce a new translation only for this one. Therefore, keeping artemisTranslate here will be a better option in my opinion 😄

Reply:
I see your point :+1: